### PR TITLE
[ADDED]About Page

### PR DESCRIPTION
**Pull request by ApoorvDubey23**
Added About Us page  #30 

https://github.com/EshaanAgg/LFXMM-Frontend/assets/132251091/fd3ed1f0-9c2f-458c-8965-3b0520ff6832

### CODE REVIEW

File: src/assets/images/AboutUs.jpg
Comment (EshaanAgg) [inline src/assets/images/AboutUs.jpg:1]: Crop the trademark. Try to use a SVG or webp if possible.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]
@ApoorvDubey23 
A couple of changes are needed 
- The find that you are using is not being used anywhere in the site and looks inconsistent. Please use the fonts and styling similar to what has been used elsewehere.
- Instead of creating a new About Us page, rename the team page to about us, and on the top of it add this static content.

File: src/assets/images/AboutUs.jpg
Comment (ApoorvDubey23) [reply src/assets/images/AboutUs.jpg:1]: Sure will try to find an SVG file , as currently the jpg file I am using is having a white background which I removed using online software , will try to find one

Comment (ApoorvDubey23) [reply src/assets/images/AboutUs.jpg:1]: Please Review the changes

Comment (ApoorvDubey23) [reply src/assets/images/AboutUs.jpg:1]: Bhaiya , mene ek mistake krdi thi , mene pull ni liya tha iss krn bhut saari gadbad ho jati site m , ek naya pull request generate kr rha with updated changes and sahi wala
Sorry for the inconvinience caused

**Pull request closed without merging**